Sources/Swift/Integrations/UserFeedback/Configuration/SentryUserFeedbackWidgetConfiguration.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+#if (os(iOS) || os(tvOS)) && !SENTRY_NO_UIKIT
+@_implementationOnly import _SentryPrivate
+import UIKit
+
+/**
+ * Settings for whether to show the widget and how it should appear.
+ */
+@objcMembers public class SentryUserFeedbackWidgetConfiguration: NSObject {
```

Comment:
`m`: Isn't this more of a button that opens the feedback form / widget. We could rename this to something like `
SentryFeedbackTriggerConfiguration` and then we could rename `triggerLabel` to just `label` and `triggerAccessibilityLabel` to just `accessibilityLabel`.

Reply:
Form and widget are two different things, see https://github.com/getsentry/sentry-cocoa/pull/4284#discussion_r1765741874

That being said we can certainly rename things to remove the `trigger` prefix from widget config properties. These were mostly copied wholesale from the javascript SDK's implementation/docs.

Reply:
> to just accessibilityLabel

`accessibilityLabel` is actually a property already on `NSObject` so we can't call it this. I went with `widgetAccessibilityLabel`
![image](https://github.com/user-attachments/assets/631426f3-2862-4fa2-aea8-3f5603b8dd1a)

Reply:
Also went with `labelText` instead of just `label`. Could also choose `title` if people like that more.